src/amo/reducers/reviews.js

```diff
@@ -209,12 +189,50 @@ export const getReviewsByUserId = (
     : null;
 };
 
+export const latestByAddonVersionKey = ({
+  userId,
+  addonId,
+  versionId,
+}: {|
+  userId: number,
+  addonId: number,
+  versionId: number,
+|}) => {
+  return `user${userId}-addon${addonId}-version${versionId}`;
```

Comment:
Do we need to read those values? If so, we could use a slash as separator and use the dash between the prefix and its value.